english/learn/learn.md

```diff
@@ -0,0 +1,5 @@
+Learning can be very enjoyable.
+If the teacher is fun.
+
+you can learn [magic](../magic/magic.md)
+or you can learn [singing](../sing/sing.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.